e2e/env.js

```diff
@@ -1,27 +0,0 @@
-module.exports = function () {
-
-  // see list of available hooks here
-  // https://github.com/cucumber/cucumber-js/blob/master/docs/support_files/event_handlers.md
-
-  this.setDefaultTimeout(400 * 1000);
```

Comment:
Hi @kahboom, I know it's too late but I need this file :)
Maybe I should rename the file to `env.ts` ?